gradle/feature.gradle

```diff
@@ -0,0 +1,28 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-kapt'
```

Comment:
kind of minor issue but having every feature implement all of these things is kind of overkill but not something you would necessarily have to worry about. not every feature needs kapt or view binding or needs to be an android module for example